Scripts/Python/ki/xKIChat.py

```diff
@@ -401,6 +404,21 @@ def SendMessage(self, message):
     ## Adds a line to the RT chat.
     def AddChatLine(self, player, message, cFlags, forceKI=True):
 
+        playerName = PtGetClientName()
+        playerNameMentionRegex = None
+
+        # Compose the regex differently depending on if player name starts and ends with a word or with special characters
+        if _STARTS_WITH_WORD_REGEX.search(playerName) != None:
```

Comment:
```suggestion
        if _STARTS_WITH_WORD_REGEX.search(playerName) is not None:
```

`is` is preferred for checking singletons like `None`.

Reply:
Updated all the occurrences I made with "!= None" to be "is not None" instead.